.secrets/.gitignore

```diff
@@ -1,6 +0,0 @@
-# IMPORTANT! This folder is hidden from git - if you need to store config files or other secrets,
```

Comment:
Is this change voluntary?

Reply:
Yes but can be removed; at our company we follow the philosophy of "only one gitignore"